mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/SmartTestingMavenConfigurer.java

```diff
@@ -28,17 +28,23 @@
 
     private Configuration configuration;
 
+    private boolean skipExtension;
+
     @Override
     public void afterProjectsRead(MavenSession session) throws MavenExecutionException {
 
         configuration = Configuration.load();
 
-        if (configuration.isDisabled()) {
+        final MavenProjectConfigurator mavenProjectConfigurator = new MavenProjectConfigurator(configuration);
```

Comment:
As I have written above - it is not necessary to create an instance of the class just because of retrieving system properties...